_posts/2019-07-19-p2p-data-sync-for-mobile.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-name:  "P2P Data Sync for Mobile"
-title:  "P2P Data Sync for Mobile"
-date:   2019-07-19 12:00:00 +0800
+name: 'P2P Data Sync for Mobile'
```

Comment:
Is there a specific reason we are changing double quotes to single quotes? As opposed to keeping the md files as they are

Reply:
Sorry, that was changed by my Prettier settings. Fixed.